### PR TITLE
Ensure release_helper.sh creates PRs on main repo

### PR DESCRIPTION
**Pull request by CoderDake**
Fixes https://github.com/flutter/devtools/issues/6282

### CODE REVIEW

File: tool/release_helper.sh [hunk 1]
```diff
@@ -59,4 +59,6 @@ echo
 
 echo "Your Draft release PR can be found at: $PR_URL"
 echo
-echo "$0 DONE: Build, run and test this release using: `dart ./tool/build_e2e.dart`"
\ No newline at end of file
+echo "$0 DONE: Build, run and test this release using: `dart ./tool/build_e2e.dart`"
+
+exit
```
Comment (polina-c) [inline tool/release_helper.sh:64]: How about adding new line to the end of file to remove this red sign?

**Review by polina-c — APPROVED** [review]

File: tool/release_helper.sh (hunk 1 above)
Comment (polina-c) [reply tool/release_helper.sh:64]: Screenshot 2023-09-05 at 8 08 38 AM

File: tool/release_helper.sh [hunk 2]
```diff
@@ -59,4 +59,6 @@ echo
 
 echo "Your Draft release PR can be found at: $PR_URL"
 echo
-echo "$0 DONE: Build, run and test this release using: `dart ./tool/build_e2e.dart`"
\ No newline at end of file
+echo "$0 DONE: Build, run and test this release using: `dart ./tool/build_e2e.dart`"
```
Comment (CoderDake) [inline tool/release_helper.sh:62]:
```suggestion
echo "$0 DONE: Build, run and test this release using: 'dart ./tool/build_e2e.dart'"
```

File: tool/release_helper.sh (hunk 1 above)
Comment (CoderDake) [inline tool/release_helper.sh:64]:
```suggestion

```

File: tool/release_helper.sh [hunk 3]
```diff
@@ -60,4 +59,5 @@ echo
 echo "$0: Your Draft release PR can be found at: $PR_URL"
 echo
 echo "$0: DONE.
-echo "$0: Next Step: Build, run and test this release using: `dart ./tool/build_e2e.dart`"
+echo "$0 DONE: Build, run and test this release using: 'dart ./tool/build_e2e.dart'"
```
Comment (CoderDake) [inline tool/release_helper.sh:62]:
```suggestion
echo "$0: Build, run and test this release using: 'dart ./tool/build_e2e.dart'"
```

**Pull request merged**